invenio_records_lom/utils/vcard.py

```diff
@@ -0,0 +1,135 @@
+            .replace(",", "\\,")
+            .replace("\n", "\\n")
+        )
+
+    @staticmethod
+    def utf_8_len(s: str) -> int:
+        """Return length of `s` in number of octets `s`'s "utf-8"-encoding has."""
+        return len(s.encode(encoding="utf-8"))
+
+    def line_wrap(self, unwrapped_vcard: str) -> str:
+        """Wrap lines as per vcard-spec.
+
+        - line-length in "utf-8"-encoding may not be longer than 75 octets (excluding newline-characters)
+        - can't break apart multi-octet-sequences when line-wrapping
+        - continued lines start with a space-character
+        """
+        input_lines = [line.lstrip() for line in unwrapped_vcard.splitlines()]
+        output_lines = []
+        for input_line in input_lines:
+            start = 0  # index where currently built line starts
+            prefix = ""  # prefix of currently built line, space for continued lines, counts towards line-length
+            length = 0  # length of currently built line, in number of "utf-8"-octets in its "utf-8" encoding
+            idx = 0  # index running through `input_line`
+            while idx < len(input_line):
+                char = input_line[idx]
+                length += self.utf_8_len(char)
+                if length > 75:
+                    # `char` would push currently built line over limit
+                    output_lines.append(prefix + input_line[start:idx])
+                    start = idx
+                    prefix = " "
+                    length = self.utf_8_len(prefix)
+                else:
+                    idx += 1
+            # handle rest of `input_line`
+            output_lines.append(prefix + input_line[start:])
+
+        return self.line_break_str.join(output_lines) + (
+            self.line_break_str if self.final_line_break else ""
+        )
+
+    def make_vcard(self, fn: str | Iterable[str]) -> str | bytes:
+        """Create a ``str``-object in `vcard`-format around passed-in `fn` (formatted name).
+
+        Names of argments are the lower-cased version of properties defined by the spec.
+        Note that all parameters can either be :str or :Iterable[str] (since same property can appear in vcard multiple times).
+
+        :param fn: formatted name, (only) required vcard-field
+        :return: built vcard, if `self.output_encoding` is truthy, encode output before return
+        """
+        if isinstance(fn, str):
+            fn = [fn]
+
+        header_lines = ["BEGIN:VCARD", "VERSION:4.0"]
+        fn_lines = [f"FN:{self.escape_noncompound_property(p)}" for p in fn]
+        footer_lines = ["END:VCARD"]
+
+        unwrapped_vcard = self.line_break_str.join(
+            header_lines + fn_lines + footer_lines
+        )
+        vcard = self.line_wrap(unwrapped_vcard)
+
+        if self.output_encoding:
+            return vcard.encode(encoding=self.output_encoding)
+        return vcard
+
+
+lom_vcard_maker = VCardMaker(
```

Comment:
opinion: I don't like constructs like that. What is wrong with defining a normal ordinary function? Like:

```
def make_lom_vcard(text: str) -> str | bytes:
    maker = VCardMaker(final_line_break=False, line_break_str="\n",output_encoding=None)
    return maker.make_vcard(text)
```